### PR TITLE
Added dynamically interactive tabs for Who am I and What Have I Made.

### PR DESCRIPTION
**Pull request by Ansem717**
Page automatically opens to the about tab on load. Didn't take too long. I ended up spending too much time without sourcing the script itself in the index.html. These silly little things will start to become first hand knowledge in no time. :P

### CODE REVIEW

File: scripts/projects.js [hunk 1]
```diff
@@ -20,24 +20,32 @@ function Project(itm){
...
   var finAuthors = '';
+  var finProj = '';
   var authorTemplate = $('#authTemp').html();
   var projectTemplate = $('#projTemp').html();
   var dataSource = this;
-  console.log(dataSource);
   var compAuth = Handlebars.compile(authorTemplate);
   var compProj = Handlebars.compile(projectTemplate);
+  //Place the heading before the first item only
+  if(inx==0){
```
Comment (bnates) [inline scripts/projects.js:33]: @Ansem717 best practice is to use the explicit '===' operator when doing explicit comparisons

File: scripts/projects.js [hunk 2]
```diff
@@ -20,24 +20,32 @@ function Project(itm){
...
     finAuthors += compAuth(ele);
     if(index!=arr.length-1){
-      finAuthors += " and ";
+      finAuthors += " and "; //Place " and " after each author that isn't the last one. We don't need to do anything for the last author.
     };
   });
   this.authorPlaceholder = finAuthors;
-  var finProj = compProj(dataSource)+"<hr/>";
-  console.log(finProj);
+  if(inx!=arry.length-1){
```
Comment (bnates) [inline scripts/projects.js:44]: @Ansem717 for style points - add a few spaces in your expressions...  this makes it easier to read/debug
example:
if ( inx !== arry.length -1 )

Additionally, be sure to use the explicit '!==' operator for comparisons - as discussed above

File: scripts/projects.js [hunk 3]
```diff
@@ -20,24 +20,32 @@ function Project(itm){
...
+    finProj += compProj(dataSource)+"<hr/>";//Place horizontal rows after each project except last
```
Comment (bnates) [inline scripts/projects.js:45]: @Ansem717 <hr /> and <br /> tags should be avoided as the stylistic (and spacing) aspect of content should managed in CSS

File: scripts/projects.js [hunk 4]
```diff
@@ -51,6 +59,6 @@ projectData.forEach(function(ele) {
   projects.push(new Project(ele));
 });
 
-projects.forEach(function(ele){
-  $('#project-placeholder').append(ele.toHtml());
+projects.forEach(function(ele, index, arr){
```
Comment (bnates) [inline scripts/projects.js:62]: @Ansem717 try to stick with semantic and similar naming conventions - here's the one's i use:

Arrays - arr, array
Element - el, ele, or elem
Index - idx, index

File: index.html [hunk 5]
```diff
@@ -13,16 +13,16 @@
   <h1>Andy Malik's Digital Portfolio</h1>
   <nav id="main-menu">
     <ul>
-      <li class="about-tab"><h2>Who am I?</h2></li>
-      <li class="project-tab"><h2>What have I made?</h2></li>
+      <li id="tab" class="about-tab" data-filter="about"><h2>Who am I?</h2></li>
+      <li id="tab" class="project-tab" data-filter="project"><h2>What have I made?</h2></li>
```
Comment (patci-zz) [inline index.html:17]: @Ansem717 Usually best practice to use a specific id no more than once.

Comment (patci-zz) [inline index.html:17]: Might help to group your li elements into a single class.

**Pull request merged**